test/call_traits_test.cpp

```diff
@@ -18,8 +18,9 @@
 #include <typeinfo>
 #include <boost/call_traits.hpp>
 
-#include <libs/type_traits/test/test.hpp>
-#include <libs/type_traits/test/check_type.hpp>
+// type_traits/test utilities
+#include <test.hpp>
+#include <check_type.hpp>
```

Comment:
This looks like it has more potential for including a wrong header, if one of these headers are present in the include paths, other than from `type_traits/test`. I think, this needs to be handled better.